packages/webpack/src/runtime/assemble.js

```diff
@@ -1,21 +1,72 @@
 const { readFileSync } = require('node:fs')
+const { parse } = require('@babel/parser')
 
 /**
  * @param {string} source
  * @returns {string}
  */
 function removeMultilineComments(source) {
-  return source.replace(/\/\*[\s\S]*?\*\//g, '')
+  try {
```

Comment:
you explained it to me, but it's worth mentioning in a comment why we need to remove multiline comments